pkgs/tools/text/rgxg/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "rgxg";
+  version = "0.1.2";
+
+  src = fetchurl {
+    url = "https://github.com/rgxg/rgxg/releases/download/v${version}/rgxg-${version}.tar.gz";
+    sha256 = "554741f95dcc320459875c248e2cc347b99f809d9555c957d763d3d844e917c6";
+  };
+
+  outputs = [ "bin" "doc" "man" "dev" "lib" "out" ];
+
+ doInstallCheck = true;
+ installCheckTarget = "check";
```

Comment:
What's wrong with regular `doCheck` and `checkTarget`?

Reply:
nothing,
i don't understand everything yet, this was copied from an other package, and i was just happy that it worked

Reply:
I see you are new :) Welcome aboard. I don't know where you borrowed that from, but it's considered more common to use `doCheck = true`. `check` is the default check (make) target for `doCheck`, so you should spare yourself 1 line of from the expression. For more info, refer to https://nixos.org/nixpkgs/manual/#chap-stdenv .

Reply:
I don't know how to edit my commits here, so i created a new pr with 2 commits:
https://github.com/NixOS/nixpkgs/pull/99422

Thanks for the weclome